blockchain/vm/contracts.go

```diff
@@ -60,9 +60,9 @@ type PrecompiledContract interface {
 	Run(input []byte, contract *Contract, evm *EVM) ([]byte, error)
 }
 
-// PrecompiledContractsCypress contains the default set of pre-compiled Klaytn
-// contracts used in the Cypress release.
-var PrecompiledContractsCypress = map[common.Address]PrecompiledContract{
+// PrecompiledContractsConstantinople contains the default set of pre-compiled Klaytn
+// contracts used in the Constantinople release.
+var PrecompiledContractsConstantinople = map[common.Address]PrecompiledContract{
```

Comment:
It's too long but it is more clear. How about this?
```suggestion
var PrecompiledContractsConstantinopleCompatible = map[common.Address]PrecompiledContract{
```

Reply:
I also considered istanbulCompatible. However the name of instructionsets follows geth, so I omit `compatible`.